tiles-generation/docker/vector-tile/modified-s5cmd/cp.go

```diff
@@ -0,0 +1,868 @@
+//  Copyright OpenSearch Contributors
+//  SPDX-License-Identifier: Apache-2.0
+
+// In order to support add metadata 'Content-Encoding=gzip' and 'Content-Type=application/x-protobuf' on copy command,
+// modified this file from https://github.com/peak/s5cmd for docker image build.
+
+package command
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"mime"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/urfave/cli/v2"
+
+	errorpkg "github.com/peak/s5cmd/error"
+	"github.com/peak/s5cmd/log"
+	"github.com/peak/s5cmd/log/stat"
+	"github.com/peak/s5cmd/parallel"
+	"github.com/peak/s5cmd/storage"
+	"github.com/peak/s5cmd/storage/url"
+)
+
+const (
```

Comment:
was there any specific reason for writing these files in go and not in some other language like python?

Reply:
Because the s5cmd written in go, we just modified some of the code to support our feature. I added the comments at the head that why we modified file. 

```
// In order to support add metadata 'Content-Encoding=gzip' and 'Content-Type=application/x-protobuf' on copy command,
// modified this file from https://github.com/peak/s5cmd for docker image build.
```